### ISSUE TITLE
Resuming playback with a button in notification does not work

### ISSUE THREAD

**Issue by eternal-sorrow** [PaulWoitaschek/Voice#1958]
### Overview

After some time when I try to resume playback with a notification, it does not work. Opening the app an pressing play works as usual, but I need to unlock the phone to do this. This is not a duplicate off #1777, because there is no need to kill and restart the app, just open it and start the playback as usual. And also this is not fixed still in the current f-droid version.

### How to reproduce

1. Listen to some book.
2. Payse the playback.
3. Wait for some time (at least 1h, mabe even more).
4. Try to resume playback with a notification.

### Version

7.0.14

### Android Version

12

### Device Model

Sony Xperia XA2

**Comment by PaulWoitaschek** [comment]
Thanks for the report! The new release should arrive within the next minutes.

**Comment by eternal-sorrow** [comment]
Yep, in 7.1.2 this is fixed. There is a small issue, though I don't know iv it is an application's fault or my android system's. When there is more than one media notification (in my example, Voice and NewPipe), I need to swipe left or right to open the Voice one. But, when I do that, the notification drawer closes. And then it keeps closing when I open it again. After some struggling I'm able to press the play button and then it works as it should (and the drawer stops closing).

**Comment by newhinton** [comment]
@PaulWoitaschek I have the same issue, however with 8.1.3. Should i create a new Ticket or expand this one with the nessessary information?

**Comment by PaulWoitaschek** [comment]
Please go with a new one :)
